### DISCUSSION TITLE
나무나 의자 등 예제에 나오는 오브젝트 질문입니다!

### DISCUSSION THREAD

**Discussion by kimyang-sun** [naverz/zepeto-studio-kor#1048] [5. Q&A]
안녕하세요! 항상 도움 많이 받고 있습니다.
다름이 아니라 멀티플레이 예제에 나오는 나무나 의자 오브젝트들도 자유롭게 사용이 가능한 오브젝트들인가요??
그 외에 예제에 나오는 다른 오브젝트들도 상업적으로도 사용이 가능한 오브젝트들인지 궁금합니다!
아래는 나무 예시 이미지입니다!

![image](https://user-images.githubusercontent.com/56675004/180167966-d4e34506-b7fe-4047-aa90-df551ffd50a9.png)

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요~ 넵 마음대로 사용하셔도 좋습니다 :)

**Reply by kimyang-sun** [reply]
항상 빠른 답변 감사합니다! :)
